src/cli.ts

```diff
@@ -101,13 +150,39 @@ export function cliWarn(message: string, fromVercelCli = false): void {
  * the function also removes extra indentation on the message allowing us to indent the messages
  * in the code as we please (see https://www.npmjs.com/package/dedent-tabs)
  */
-function prepareCliMessage(message: string, fromVercelCli: boolean): string {
-	return dedent(message)
+function prepareCliMessage(
+	message: string,
+	{ fromVercelCli, styleFromater, spaced }: PrepareCliMessageOptions
+): string {
+	const preparedMessage = dedent(message)
 		.split('\n')
-		.map(line => `${getCliPrefix(fromVercelCli)} ${line}`)
+		.map(
+			line =>
+				`${getCliPrefix(fromVercelCli)} ${
+					styleFromater ? styleFromater(line) : line
+				}`
+		)
 		.join('\n');
+
+	return spaced ? `\n${preparedMessage}\n` : preparedMessage;
 }
 
 function getCliPrefix(fromVercelCli: boolean): string {
 	return fromVercelCli ? '▲' : '⚡️';
```

Comment:
```suggestion
	return fromVercelCli ? '▲ ' : '⚡️';
```
sorry this is unrelated to your changes but could you add a space here so that we align the logs (it seems like the lightning emoji adds an extra space 😓)

![Screenshot 2023-04-17 at 12 18 58](https://user-images.githubusercontent.com/61631103/232469955-8f2b107f-04ed-4d5e-a829-cac9c3936518.png)

Reply:
Thanks 🙂 

but dang, in your terminal the lightning emoji is so misaligned 😓 , well a problem for another time 🤷